Add options to specify doc build target

denisok opened the pull request:
PR is needed to be able to build html or/and pdf

Commit:
Add options to specify doc build target

it-percona-cla commented:
[![CLA assistant check](https://cla-assistant.percona.com:443/pull/badge/signed)](https://cla-assistant.percona.com:443/percona/percona-toolkit?pullRequest=551) <br/>All committers have signed the CLA.

nastena1606 commented:
Hi! Thanks for the PR. I've tested the build with --html flag but the script still builds both html and pdf docs.

Commit:
Fix getopt to work on Mac, which has olf unix version. Add new option

Commit:
fix debug

denisok commented:
@nastena1606 you are using Mac that has old `getopt` unix version, which doesn't support long options. I changed it to short one, please try again. Also added `-t` for your request to have new make option in a future for new theme.

nastena1606 reviewed (approved):
LGTM, thank you!
